docs/1.0. Overview.md

```diff
@@ -45,40 +45,38 @@ Note that:
 The above Map is represented in JSON in the API as below. See [Behaviour](4.0.%20Behaviour.md) for more details.
 
 ```json
-"map":{
-  "Output A":{
-    "chan 1" :{
-      "input": "Input A",
+"map": {
+  "OutputA": {
+    "0": {
+      "input": "InputA",
       "channel_index": 0
     },
-    "chan 2":{
-      "input": "Input A",
+    "1": {
+      "input": "InputA",
       "channel_index": 1
     },
-    ...
+    …
```

Comment:
I think perhaps the red-backed ellipsis (here and repeated twice just below) looks more off-putting than the three separate dots?